index.js

```diff
@@ -7,12 +7,128 @@ function ask(questionText) {
   });
 }
 
-start();
+//random number generator
+function randomInt(maxNum, minNum) {
+  return Math.floor(minNum + Math.random() * (maxNum - minNum + 1));
+}
+
+// calculate guessed number
+function calcInt(maxNum, minNum) {
+  return Math.floor(((maxNum - minNum) / 2) + minNum)
+}
+
+go();
+// initialize global game variables
+let highNum = 5;
+let lowNum = 1;
+let humanAnswer = "";
+let numberOfTries = 0
+let humanTricks = 0
+let newGuessed = 0
+
+
+
+async function go() {
+  console.log(
+    "\nLet's play a game where you (human),\npick a secret number between 1 and 100,\nand I (computer) try to guess it.\nOK?\n"
+  );
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+
+  // create game loop OR error out if non number, decimal number or number out of range is entered
+  if ((secretNumber % 1 === 0) && (secretNumber >= lowNum && secretNumber <= highNum)) {
+    console.log("\nYou entered: " + secretNumber + "\n");
```

Comment:
Debug statements are good, but remember to remove them when you're done with them to keep your code clean, and readable.